test/JDBC/input/BABEL-733-vu-prepare.mix

```diff
@@ -0,0 +1,44 @@
+-- tsql
```

Comment:
these tests should be added to upgrade tests, add them to latest and 14_6/14_7 schedule file

Reply:
Added in latest commit.